frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -2878,6 +2878,86 @@
             }
         };
 
+        vm.editchallengeTagDialog = function(ev) {
+            vm.tags = [];
+            for (var i = 0; i < vm.page.list_tags.length; i++) {
```

Comment:
use slice instead of for loop

Reply:
Yes we can assign it that way, i was trying to implement it with clickable button first so i assigned it that way.